langworld_db_data/movers/listed_value_mover.py

```diff
@@ -0,0 +1,48 @@
+from pathlib import Path
+
+from langworld_db_data.adders.listed_value_adder import ListedValueAdder
+from langworld_db_data.constants.literals import ID_SEPARATOR
+from langworld_db_data.filetools.csv_xls import read_dicts_from_csv, write_csv
+from langworld_db_data.movers.mover import Mover, MoverError
+from langworld_db_data.removers.listed_value_remover import ListedValueRemover
+
+
+class ListedValueMoverError(MoverError):
+    pass
+
+
+class ListedValueMover(Mover):
+    def __init__(  # type: ignore
+        self,
+        **kwargs,
+    ):
+        super().__init__(**kwargs)
+        self.listed_value_adder = ListedValueAdder(
+            input_file_with_listed_values=self.output_file_with_listed_values,  # Otherwise in tests
+            # listed_value_adder opens the input inventory where the removed value is
+            # present and throws an error
```

Comment:
it's not a good idea to assign a value
so that the tests run OK.
You should instantiate a `ListedValueMover` in tests
with values appropriate for tests

Reply:
Done! However I am afraid that my implementation might use practices that are not considered best ones